client/src/Component/MainTable.js

```diff
@@ -34,86 +40,146 @@ const MainTable = (props) => {
 						<th scope="col">Booking status</th>
 						<th scope="col">Pick up date</th>
-						<th scope="col">Residency status</th>
-						<th scope="col">Country of origin</th>
-						<th scope="col">Time in Scotland</th>
-						<th scope="col">Language spoken</th>
-						<th scope="col">English speaker</th>
-						<th scope="col">English skill level</th>
-						<th scope="col">Gender</th>
-						<th scope="col">DOB</th>
-						<th scope="col">Postcode</th>
-						<th scope="col">Referring agency</th>
 					</tr>
 				</thead>
 				<tbody>
-					{props.clientData.map((client) => {
+					{props.clientData.map((client, i) => {
 						return (
-							<tr key={client.client_id}>
-								<td>
-									<Checkbox
-										isChecked={props.selectedClients.includes(client.client_id)}
-										label={client.name}
-										onChange={(checked) => props.onHandleSelectedUserState(checked, client.client_id)}
-									/>
-								</td>
-								<td>{client.client_id}</td>
-								<td>{new Date(client.date_added).toLocaleDateString()}</td>
-								<td className="fw-bold">{client.name}</td>
-								<td>{client.phone_number}</td>
-								<td className="align-bikes-needed">{client.bikes_needed}</td>
-								<td className="fs-5"><span className={bookingStatusStyle[client.booking_status]}>{bookingStatus[client.booking_status]}</span></td>
-								<td>{validDate(client.pick_up_date)}</td>
-								<td>{residencyStatus[client.residency_status]}</td>
-								<td>{client.country_of_origin}</td>
-								<td>{client.time_in_scotland}</td>
-								<td>{client.language_spoken}</td>
-								<td>{speakEnglish(client.english_speaker)}</td>
-								<td>{client.english_skill_level}</td>
-								<td>{client.gender}</td>
-								<td>{validDate(client.date_of_birth)}</td>
-								<td>{client.postcode}</td>
-								<td>{client.referring_agency}</td>
-							</tr>
+							<Fragment key={client.client_id}>
+								<tr>
+									<td className="">
+										<Checkbox
+											isChecked={props.selectedClients.includes(
+												client.client_id
+											)}
+											label={client.name}
+											onChange={(checked) =>
+												props.onHandleSelectedUserState(
+													checked,
+													client.client_id
+												)
+											}
+										/>
+									</td>
+									<td>{client.client_id}</td>
+									<td>{new Date(client.date_added).toLocaleDateString()}</td>
+									<td className="fw-bold d-flex justify-content-start align-items-center">
+										<div className="fs-4 me-3">
+											<svg
+												xmlns="http://www.w3.org/2000/svg"
+												width="16"
+												height="16"
+												fill="currentColor"
+												viewBox="0 0 16 16"
+												onClick={() => {
+													props.showOnClientInfoClick(i);
+												}}
+												cursor="pointer"
+											>
+												{props.isHiden === i
+													? arrowPositionDown
+													: arrowPositionUp}
+											</svg>
```

Comment:
```suggestion
											</svg>
											<span className="visually-hidden">
												{props.isHiden === i
													? "Show less details"
													: "Show more details"}
											</span>
```
The button doesn't have any visible text, but we need to give it text so that screen readers can read out text for the button. We can add button text that is visually hidden that says `Show more details` or `Show less details` depending on the state of the button

Reply:
That is a great idea and good practice. thank you so much.